docs/source/API/core/utilities/all.md

```diff
@@ -0,0 +1,25 @@
+(KokkosALL)=
+# `Kokkos::ALL`
+
+Defined in header `<Kokkos_Core.hpp>`
+
+```c++
+namespace Kokkos{
+  KokkosInternalType ALL = impl_detail;
+}
+```
+
+Type used as tag to specify the selection of all elements in a dimension.
```

Comment:
`Kokkos::ALL` is a constant of unspecified type that is used to ...

Reply:
done in [56bb6e](https://github.com/kokkos/kokkos-core-wiki/pull/132/commits/56bb6edb1d564172c365d27c48249968751039b0)